apps/users/tests/test_forms.py

```diff
@@ -179,6 +179,16 @@ def test_no_names(self):
         r = self.client.post(self.url, data)
         self.assertFormError(r, 'form', 'username', 'This field is required.')
 
+    def test_fxa_id(self):
+        assert self.user.fxa_id is None
+        data = {'username': 'blah',
+                'email': 'jbalogh@mozilla.com',
+                'fxa_id': 'yo',
+                'lang': 'en-US'}
+        r = self.client.post(self.url, data)
+        assert r.status_code == 302
```

Comment:
would this benefit from `assertRedirect()` maybe? I don't fully understand what this test is catching.